src/main/java/org/dcsa/ebl/model/transferobjects/TransportDocumentTO.java

```diff
@@ -79,4 +80,6 @@ public class TransportDocumentTO {
   @Valid private List<TransportTO> transports;
 
   @Valid private List<ShipmentLocationTO> shipmentLocations;
+
+  @Valid private PartyTO party;
```

Comment:
variable should be named issuingParty according to the OAS

Reply:
Good point. Renamed.